hash/apple2_flop_clcracked.xml

```diff
@@ -41013,6 +40652,41 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="animate">
+		<description>Animate (4am crack)</description>
+		<year>1986</year>
+		<publisher>Broderbund Software</publisher>
```

Comment:
I think people have started changing these to say “Brøderbund” as it’s usually rendered on-screen

Reply:
@cuavas - I accidentally caused the “Brøderbund” issue.  I have fixed it.  I am going to close this pull request since it's been in the queue for so long.  I will be wrapping up a new pull request this weekend based on the current Mame source.